src/main/java/sirius/biz/protocol/AuditLog.java

```diff
@@ -89,6 +88,33 @@ public AuditLogBuilder forUser(@Nullable String id, @Nullable String name) {
             return this;
         }
 
+        /**
+         * Marks the entry as caused by the current user in {@link UserContext#getCurrentUser()}.
+         *
+         * @return the builder for fluent method calls
+         */
+        @CheckReturnValue
+        public AuditLogBuilder causedByCurrentUser() {
+            UserInfo user = UserContext.getCurrentUser();
+            return causedByUser(user.getUserId(), user.getUserName()).forTenant(user.getTenantId(),
```

Comment:
so this could overwrite the tenant set in forcurrentuser? isn't this wrong as the tenant should be the tenant of the user to log for?